packages/x-data-grid/src/components/cell/GridCell.tsx

```diff
@@ -75,9 +74,7 @@ export type GridCellProps = {
   [x: string]: any; // TODO v7: remove this - it breaks type safety
 };
 
-type CellParamsWithAPI = GridCellParams<any, any, any, GridTreeNodeWithRender> & {
-  api: GridApiCommunity;
-};
+type CellParamsWithAPI = GridCellParams<any, any, any, GridTreeNodeWithRender>;
```

Comment:
Not sure this type name makes sense anymore, given GridCellParams has `api` - can we just use `GridCellParams` and remove this type?

Reply:
Thanks for the review,
can we just change the name of this type to let's say `CellParams` ? because we are using this type at one more place and ` GridCellParams<any, any, any, GridTreeNodeWithRender>` seems long to me .

Reply:
we should also probably change the name of this variable to `cellParams` ?
https://github.com/mui/mui-x/blob/496d64eac8400f6f06ba94886b7ae3695a7303c8/packages/x-data-grid/src/components/cell/GridCell.tsx#L186

Reply:
Both of those suggestions make sense to me 👍

Reply:
we can remove the type I think  did a little refactoring  in [935aed2](https://github.com/mui/mui-x/pull/14201/commits/935aed229bcbd2a7ea359b358edf7f7719da3e0f)